pkg/wolfssl/Makefile.dep

```diff
@@ -82,3 +82,6 @@ endif
 ifneq (,$(filter wolfcrypt_random,$(USEMODULE)))
   USEMODULE += random
 endif
+
+# wolfssl is only supported by 32 bit architectures
+FEATURES_REQUIRED += arch_32bit
```

Comment:
@Rotzbua is right. This package needs a 

```
FEATURES_BLACKLIST += arch_esp8266
```

Reply:
The comment was only saying "only 32 bit architectures". Do we know why esp8266 boards were blacklisted ?

Reply:
Sadly, no. Weirdly enough, in https://github.com/RIOT-OS/RIOT/pull/12454 the ESP8266 wasn't a problem. Maybe @danielinux, @maribu, @MichelRottleuthner, or @smlng could give some insights?

Reply:
and #10308 doesn't give more info unfortunately.

Reply:
Maybe it was just blacklisted by accident or @danielinux had problems with it? I don't remember discussing any specific problems of wolfssl on the esp but also I didn't test on that platform.